### ISSUE TITLE
Enhancement: update OpenShift routes "status" to update console display 

### ISSUE THREAD

**Issue by chen23** [F5Networks/k8s-bigip-ctlr#477]
#### Description
In OpenShift the GUI console relies on metadata in the route entry to determine if a route has been "admitted".

#### Kubernetes Version
OpenShift Origin 3.7

#### Controller Version
1.3.0

#### BIG-IP Version
12.1.1

#### Diagnostic Information

Example w/out status

![f5-container-connector-routes-no-status](https://user-images.githubusercontent.com/63689/34826672-7cf834f8-f6a5-11e7-9e1c-ef4366a6d65a.png)

Sample JSON added via something like:
```
curl -s -X PUT http://localhost:8001/oapi/v1/namespaces/default/routes/f5-demo-app/status --data-binary @./modified-f5-demo-app.json -H Content-Type:application/json 
```
Used the following for reference: https://github.com/openshift/origin/blob/master/images/router/clear-route-status.sh
```
{
  "status": {
    "ingress": [
      {
        "conditions": [
          {
            "type": "Admitted",
            "status": "True"
          }
        ],
        "wildcardPolicy": "None",
        "host": "f5-demo-app-default.apps.f5demo.com",
        "routerCanonicalHostname": "bigip-dns-wideip1.f5demo.com",
        "routerName":"bigip1"
      }
    ]
  }
}
```

Example w/ status

![f5-container-connector-routes-with-status](https://user-images.githubusercontent.com/63689/34826680-85dd2312-f6a5-11e7-8b82-c5ab775df494.png)

**Comment by palonsoro** [comment]
Hi,

This feature is very important. Without it, users do not have any feedback about whether the route is being exposed on F5 or not.

Furthermore, as you can see in captures, if the status is nor properly set **as OpenShift expects**, it shows a false message about OpenShift being unable to expose the route. From my point of view, it makes the problem described at the issue a bug, not an enhancement, because a confusing "not working" message is shown to users regardless of whether the route is actually working or not and the only reason for it to happen is that Big IP controller is not respecting OpenShift conventions.

**Comment by mhermsdorfer** [comment]
Bumping to express how critical this is.  I agree with @palonsoro this feels much more like a bug than a feature enhancement as a typical user would be lead to believe that the F5 is non-operational, even when it is correctly functioning.

**Comment by jackf242** [comment]
+1 @mhermsdorfer

**Comment by mdditt2000** [comment]
https://jira.pdsea.f5net.com/browse/CONTCNTR-246
https://jira.pdsea.f5net.com/browse/CONTCNTR-304

Requirement for OpenShift 4.1 and 3.11

**Comment by mdditt2000** [comment]
On target for CIS 1.11 coming early Oct

**Comment by mdditt2000** [comment]
@chen23 i have a early image if you want to validate. Let me know.

**Comment by chen23** [comment]
yes pls.

**Comment by mdditt2000** [comment]
@chen23 resolved for OCP 3.11 and OCP 4.1

OCP 3.11 is in CIS 1.11 and OCP 4.1 will be in the CIS 1.11.1 patch release

![image](https://user-images.githubusercontent.com/12733938/66801629-56340400-ef37-11e9-8730-0ed982091127.png)

Closing this issue out
